README.md

```diff
@@ -1,118 +1,47 @@
-# Template repository for a Next.js application
+# Template repository for application infrastructure
 
-This is a template repository for a Next.js-based application. While this template should be opinionated in some ways in order to reduce setup overhead where possible, it should be application-agnostic, meaning that any type of Next.js application should be able to be created from this template.
+## Overview
 
-## Contents
+This is a template repository for a to set up foundational infrastructure for your application in AWS. This template includes setup for:
 
-This template includes setup for:
+* **Team workflows** - PR template, architecture decision record (ADR) template, Makefile.
+* **Infrastructure as code** - infrastructure for terraform backends, including S3 buckets and DynamoDB table for locking terraform state
+* **CI for infra** - GitHub action that performs infra code checks, including linting, validation, and security compliance checks.
+* **Application infrastructure** - the infrastructure you need to set up a basic web app, such as a image container repository, load balancer, web service, and database.
+* **CD / Deployments** - infrastructure to set up AWS account access for GitHub actions, Makefile commands for building and publishing release candidates, and a GitHub action for deploying on merges to main.
```

Comment:
Could this maybe be summarized a bit before the long list? Or maybe it's just that the "continuous deployment" part is shortened in the title.

```suggestion
* **CD / Deployments** - infrastructure for continuous deployment, including: AWS account access for Github actions, scripts for building and publishing release artifacts, and a Github action for automated deployments from the main branch.
```